### PR TITLE
Remove final keyword from RedisCache

### LINKED ISSUE #6: RedisCache:get
Hey there,

So i've been investigating what is wrong with my project and i've encountered a raise condition in your library.
1.  `RedisCache:get` method uses not only `get` but also `exists` so that's 2 calls to database instead of 1.
2. There is raise condition here, and i've encountered it. If you call `exists` and then before you send a `get` TTL of this keywords expires, you are screwed.

I'm willing to do a pull request if needed for this one.
Also i did pull request so i can fix this issue at least on my side without `final` keyword.

### PR DESCRIPTION
**Pull request by arnaspet**
Fixes #6 

#### What does this PR do?
Well, first of all hi again, im still using your library and needed few bugfixes, and i encouter a final keyword. I dont think there should be `final` keywords in libraries at all. Especially in classes which should be used outside your library ( not internal ones ).

### CODE REVIEW

**Comment by arnaspet** [comment]
@chadicus  ?

File: src/RedisCache.php [hunk 1]
```diff
@@ -10,7 +10,7 @@
 /**
  * A PSR-16 implementation which stores data in a RedisDB collection.
  */
-final class RedisCache implements CacheInterface
+class RedisCache implements CacheInterface
```
Comment (chadicus) [inline src/RedisCache.php]: What is the reasoning for the removal of the final keyword?

**Comment by chadicus** [comment]
If you remove the commit which removes the `final` keyword, i can get this merged

**Review by chadicus — CHANGES_REQUESTED** [review]
Please remove the commit which removes the `final` keyword.

**Comment by chadicus** [comment]
@arnaspet the removal of `exists` functionality was merged with PR #8  and released with v0.1.1.  If you want to discuss the removal of the `final` keyword, please open a new issue.

**Pull request closed without merging**